ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -26,13 +45,91 @@ class SignalSpec: QuickSpec {
 			pending("should deallocate after completing") {
 			}
 
-			pending("should forward events to observers") {
-			}
-
-			pending("should dispose of returned disposable upon error") {
-			}
-
-			pending("should dispose of returned disposable upon completion") {
+			it("should forward events to observers") {
+				let numbers = [ 1, 2, 5 ]
+				
+				let signal: Signal<Int, NoError> = Signal { observer in
+					testScheduler.schedule {
+						for number in numbers {
+							sendNext(observer, number)
+						}
+					}
+					return nil
+				}
+				
+				var fromSignal: [Int] = []
+				
+				signal.observe(
+					next: { number in
+						fromSignal.append(number)
+					},
+					error: { _ in },
+					completed: {}
```

Comment:
@kastiglione yep seen that, though didn't know you can just left them out. Removed now.